### PR TITLE
Mental health hospitalizations

### PR DESCRIPTION
**Pull request by floresj**
Epics #236, #235 
Issues: #846, #847

Added Hospitalization sub-section.

### CODE REVIEW

File: src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  render () {
+    return (
+      <div className="hospitalization">
+        <div className="eapp-field-wrap">
+          <h3>{i18n.t(`psychological.hospitalization.heading.admission`)}</h3>
+          <RadioGroup className="admission" name="admission" selectedValue={this.props.Admission}>
+            <Radio
+              className="voluntary-option"
+              value="Voluntary"
+              onUpdate={this.updateAdmission.bind(this, 'Voluntary')}>
```
Comment (bmallred) [inline src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx]: Is the binding here necessary? From the `Radio` the `onUpdate` receives a payload of:

```json
{
  name: this.props.name,
  value: this.state.value,
  checked: this.state.checked
}
```

so couldn't `updateAdmission (values)` leverage something in that?

Comment (floresj) [inline src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx]: Ahh lame I thought I removed that. :+1:

File: src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+              <div className="involuntary">
+                {i18n.m('psychological.hospitalization.label.involuntaryAdmission')}
+              </div>
+            </Radio>
+          </RadioGroup>
+        </div>
+
+        <Show when={this.props.Admission}>
+          <div className="eapp-field-wrap no-label">
+            <h3>{i18n.t(`psychological.hospitalization.heading.treatment`)}</h3>
```
Comment (bmallred) [inline src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx]: An `h3` in the `eapp-field-wrap`? I've been putting them outside of the wrapping to ensure nothing is applied to them.

File: src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+            />
+            <HelpIcon />
+          </Help>
+        </div>
+
+          <div className="eapp-field-wrap">
+            <h3>{i18n.t(`psychological.hospitalization.heading.address`)}</h3>
+            <Help id={ `psychological.hospitalization.help.address` }>
+              <Address name="FacilityAddress"
+                bind={true}
```
Comment (bmallred) [inline src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx]: `bind` is probably not necessary here.

Comment (floresj) [inline src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx]: :+1:

File: src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx (hunk 2 above)
Comment (floresj) [inline src/components/Section/Psychological/Hospitalizations/Hospitalization.jsx]: :+1:

File: src/config/locales/en.js [hunk 4]
```diff
@@ -4193,12 +4193,14 @@ const en = {
       intro: 'Why we are asking',
       competence: 'Has a court or administrative agency ever issued an order declaring you mentally incompetent?',
       consultation: 'Has a court or administrative agency EVER ordered you to consult with a mental health professional',
-      consultation2: '**For example, a psychiatrist, psychologist, licensed clinical social worker, etc.**\n\n(An order to a military member by a superior officer is not within the scope of this question, and therefore would not require an affirmative response. An order by a military court would be within the scope of the question and would require an affirmative response)'
+        consultation2: '**For example, a psychiatrist, psychologist, licensed clinical social worker, etc.**\n\n(An order to a military member by a superior officer is not within the scope of this question, and therefore would not require an affirmative response. An order by a military court would be within the scope of the question and would require an affirmative response)',
```
Comment (bmallred) [inline src/config/locales/en.js]: Is the leading white space off on this or is it just the Github viewer?

Comment (bmallred) [inline src/config/locales/en.js]: Is the leading whitespace off?

Comment (floresj) [inline src/config/locales/en.js]: Hmmmm I just noticed when I edit that file it's doing a weird indentation. I'll have to check out my config for that

**Pull request merged**